R/community.R

```diff
@@ -2782,3 +2782,44 @@ communities <- groups.communities
 #' @export
 #' @family functions for manipulating graph structure
 contract <- contract_vertices_impl
+
+
+#' Voronoi partitioning of a graph
+#'
+#' @description
+#' `r lifecycle::badge("experimental")`
+#'
+#' This function partitions the vertices of a graph based on a set of generator
+#' vertices. Each vertex is assigned to the generator vertex from (or to) which
+#' it is closest.
+#'
+#' [groups()] may be used on the output of this function.
+#'
+#' @param graph The graph to partition into Voronoi cells.
+#' @param generators The generator vertices of Voronoi cells.
+#' @inheritParams distances
+#' @param mode Character string. In directed graphs, whether to compute
+#'   distances from generator vertices to other vertices (\dQuote{out}), to
```

Comment:
instead of dQuote you can use a back tick.  https://roxygen2.r-lib.org/articles/rd-formatting.html#code

```suggestion
#'   distances from generator vertices to other vertices (`out`), to
```